docs/STM32_DEVELOPMENT.md

```diff
@@ -0,0 +1,49 @@
+# mLRS Documentation: Building and Flashing for STM32 Devices #
+
+([back to main page](../README.md))
+
+mLRS uses STM32CubeIDE for STM32 targets. The following procedure should work.
```

Comment:
Nitpick - the title says '...for STM32 Devices' but here says '...for STM32 targets' don't really prefer one or other but think that they should be consistent.

Reply:
hm ... interesting. It's btw so also for the ESP page.
I see your point, but somehow I dont like targets in the title. It's really no argument, but "devices" sounds more like the actual thing and also a bit more general to me.
maybe we could change the first sentence in some way

Reply:
Let's not overthink then - will merge.